test/blackbox-tests/test-cases/melange/reason.t/dune-project

```diff
@@ -0,0 +1,2 @@
+(lang dune 3.5)
+(using melange 0.1)
```

Comment:
if you define the package in the project file, you can skip the empty opam file.

Reply:
https://github.com/jchavarri/dune/pull/5/commits/11bd20d4edd95b93acca0d7ea1a8e0f6e7608df2